contracts/abstract/Version.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
```

Comment:
spotted there are other abstract contracts in respective directory - would this be better in a `base` or `version` directory